playbooks/ansible/roles/sit.gpfs/tasks/common/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install required packages
+  dnf:
+    name:
+      - unzip
+      - python3
+      - kernel-devel
+      - kernel-headers
+      - gcc
+      - gcc-c++
+      - elfutils
+      - elfutils-devel
+      - make
+    state: latest
```

Comment:
Aren't these better suited as _server_ tasks than _common_?

Reply:
Yes. I confused the client nodes as the nodes accessing the backend, which require these components, but here "client" means Samba client, so it's backend agnostic and only requires samba client components. I'll change that.